### PR TITLE
Update Django and other dependecies

### LINKED ISSUE #6: Django 1.6
https://docs.djangoproject.com/en/1.6/releases/1.6/

### PR DESCRIPTION
**Pull request by peterhgombos**
Fixes #6

### CODE REVIEW

File: memeblast/settings/settings.py.example [hunk 1]
```diff
@@ -161,3 +159,5 @@ LOGGING = {
         },
     }
 }
+
+TEST_RUNNER = 'django.test.runner.DiscoverRunner'
```
Comment (sigvef) [inline memeblast/settings/settings.py.example]: Newline at the end of the file, please.

**Comment by peterhgombos** [comment]
Updated

**Comment by sigvef** [comment]
lgtm :+1:

**Comment by stianjensen** [comment]
Should migrations be updated to 1.7 format?

**Comment by cristeahub** [comment]
@stianjensen Yes, when updating to django 1.7 it is courtesy to also update the migrations. Also the apps should have a new structure within for instance the `__init__.py`

**Comment by stianjensen** [comment]
What should be done with the __init__.py?

**Comment by peterhgombos** [comment]
What migrations?

**Comment by stianjensen** [comment]
https://github.com/ninjadev/blast-server/tree/master/memeblast/feed/migrations

**Comment by peterhgombos** [comment]
Yeah, I have no idea how that works.

**Comment by stianjensen** [comment]
I can do it

**Comment by cristeahub** [comment]
@stianjensen : like this (not exactly like this, but this name thing):
```
# thing/__init__.py
default_app_config = 'projectname.apps.tihings.apps.ThingConfig'
```

**Comment by stianjensen** [comment]
@cristeahub Okay, is this important? And is that line the only thing you need to add?
I found this based on what you said, but that didn't really explain why we need it:
https://docs.djangoproject.com/en/1.7/ref/applications/

**Comment by stianjensen** [comment]
I added new migrations. Make sure to delete all *.pyc files in the memeblast/feed/migrations/ folder.

**Comment by sigvef** [comment]
Regarding the app config issue raised by @cristeahub: it is a workaround to properly support multiple apps with similar names, e.g. `django.contrib.auth` and a hypothetical `blast.auth`. The Django application registry introduced in 1.7 has problems with differentiating apps with the same name, but in different paths, since the label used internally in the registry to identify app is automatically generated from the module name. The workaround is to create an explicit label specified through an AppConfig (i.e., a class inheriting `from AppConfig` in `apps.py`). This label should be something that is more unique, such as for instance the entire module path, e.g. `blast.auth`. However, there is a final caveat with this, which is that the app registry doesn't play well with labels that contain periods[1], so a different convention such as `blast__auth` is preferable.

[1]: Or, at least they didn't in Django 1.7.1.

**Comment by stianjensen** [comment]
Okay. Is the convention as of 1.7 to always use the AppConfig, or only when name conflicts occur?

**Comment by sigvef** [comment]
I'm not aware of any community-wide conventions (which I guess means that the convention is to not specify labels unless there is a conflict).

**Comment by stianjensen** [comment]
Then I'd say this is Ready™

**Pull request merged**